Java CRONS wrapper docs (#8444)
getsentry/sentry-docs: src/platform-includes/crons/setup/java.mdx

Check-in monitoring allows you to track a job's progress by completing two check-ins: one at the start of your job and another at the end of your job. This two-step process allows Sentry to notify you if your job didn't start when expected (missed) or if it exceeded its maximum runtime (failed).

### Wrapper

> We offer a helper that takes care of wrapping your code and sending a check-in at the start as well as one at the end. It checks for exceptions and sets the status of the check-in accordingly.

Reviewer: It can be confusing for readers to refer Sentry as "we".

```suggestion
The Sentry Java SDK offers a helper that takes care of wrapping your code and sending a check-in at the start and one at the end. It checks for exceptions and sets the status of the check-in accordingly.
```
